### DISCUSSION TITLE
Whisper medium performing differently through cli and library

### DISCUSSION THREAD

**Discussion by swap-nil7** [openai/whisper#610] [Show and tell]
We tried to use whisper-medium for our use case since when tested on cli it was giving very nice results. We integrated it using the library as shown

```
import whisper

model = whisper.load_model("base")
result = model.transcribe("audio.mp3")
print(result["text"])
```

But now the results are not the same, please let us know if the results are different through cli and library approaches

**Comment by glangford** [comment]
See https://github.com/openai/whisper/discussions/177
